src/block-management/block-holder.js

```diff
@@ -199,12 +200,24 @@ export class BlockHolder extends React.Component<PropsType, StateType> {
 		const borderColor = isSelected ? focusedBorderColor : 'transparent';
 
 		return (
-			<TouchableWithoutFeedback onPress={ this.onFocus } >
+			// Needs to be false to access children
+			// https://facebook.github.io/react-native/docs/accessibility#accessible-ios-android
+			<TouchableWithoutFeedback
+				accessible={ false }
```

Comment:
I'm still confused by the need to set `accessible={ false }` on any element. I know why it's done here but, it feels that we're doing something wrong with our element hierarchies. @JavonDavis can you open a ticket to revisit all instances of `accessible={ false }`?

Reply:
I can understand the confusion... When creating custom elements that combine many child elements _it makes sense_ to set it to false in the sense that the default behaviour is once the element is accessible is true iOS will treat that element as a single thing vs if you needed the children to be accessible individually you'd need to counter that default behaviour

> can you open a ticket to revisit all instances of accessible={ false }?

Sure no problem!